src/axom/sidre/core/Group.cpp

```diff
@@ -1443,61 +1443,55 @@ void Group::load(const std::string& path,
   {
     SLIC_ERROR("Invalid protocol " << protocol << " for file load.");
   }
+}
+
+/*
+ *************************************************************************
+ *
+ * Load Group (including Views and child Groups) from a file
+ *
+ *************************************************************************
+ */
+Group* Group::loadChild(const std::string& path,
+                        std::string & new_name,
+                        const std::string& protocol,
+                        bool preserve_contents)
+{
+  std::string tempname = getUniqueGroupName(new_name);
+  Group * child = createGroup(tempname);
+  child->load(path, new_name, protocol, preserve_contents);
 
   if (!new_name.empty())
   {
-    renameOrWarn(new_name);
+    tempname = getUniqueGroupName(new_name);
+    child->rename(tempname);
```

Comment:
Should there be a warning if getUniqueGroupName() has changed a name due to the input name or loaded name being already in use?

Reply:
I removed this line.